core/storage/trie/impl/topper_trie_batch_impl.cpp

```diff
@@ -67,8 +67,8 @@ namespace kagome::storage::trie {
             })) {
       return false;
     }
-    /// TODO(Harrm) consider clearPrefix here. Not an easy thing and is barely
-    /// possible to happen, so leave it for the future
+    /// TODO(Harrm) PRE-462 consider clearPrefix here. Not an easy thing and is
+    /// barely possible to happen, so leave it for the future
```

Comment:
Yeah, it's actually meant to be detected by Doxygen, AFAIK it generates TODO lists from them, but as I see now it requires /// \todo rather than just TODO. Will raise up this topic in the chat or on a meeting.

Reply:
Great!